package.json

```diff
@@ -7,80 +7,76 @@
     "postinstall": "./postinstall.sh"
   },
   "devDependencies": {
-    "rn-nodeify": "^6.0.1",
-    "sleep": "^5.1.1"
+    "rn-nodeify": "^8.2.0"
   },
   "dependencies": {
-    "assert": "^1.4.1",
-    "awesome-phonenumber": "^1.0.13",
-    "babel-core": "6.24.1",
-    "babel-generator": "6.24.1",
-    "babel-helper-builder-react-jsx": "6.18.0",
-    "babel-plugin-transform-es2015-block-scoping": "6.15.0",
-    "babel-plugin-transform-es2015-parameters": "6.18.0",
-    "babel-plugin-transform-flow-strip-types": "6.18.0",
-    "babel-plugin-transform-regenerator": "6.20.0",
-    "babel-preset-react-native": "1.9.0",
-    "babel-register": "6.18.0",
+    "assert": "1.4.1",
+    "asyncstorage-down": "4.0.1",
+    "awesome-phonenumber": "2.1.2",
+    "babel-preset-react-native": "4.0.0",
     "bignumber.js": "github:status-im/bignumber.js#master",
     "browserify-zlib": "^0.1.4",
     "buffer": "^3.6.0",
-    "chance": "1.0.4",
-    "console-browserify": "^1.1.0",
+    "chance": "1.0.12",
+    "console-browserify": "1.1.0",
     "constants-browserify": "0.0.1",
-    "dns.js": "^1.0.1",
-    "domain-browser": "^1.1.7",
+    "create-react-class": "15.6.2",
+    "dns.js": "1.0.1",
+    "domain-browser": "1.1.7",
     "eccjs": "0.3.1",
-    "emojilib": "^2.2.1",
-    "events": "^1.1.1",
-    "homoglyph-finder": "^1.1.1",
+    "emojilib": "2.2.9",
+    "events": "1.1.1",
+    "homoglyph-finder": "1.1.1",
     "https-browserify": "0.0.1",
     "identicon.js": "github:status-im/identicon.js",
-    "instabug-reactnative": "git+https://github.com/status-im/instabug-reactnative.git",
-    "nfc-react-native": "github:status-im/nfc-react-native",
+    "instabug-reactnative": "git+https://github.com/janherich/instabug-reactnative.git",
```

Comment:
Comment appeared as outdated, not sure why. https://github.com/status-im/status-react/pull/2217/commits/54f5bcbe5bd8bb28ed3157c22cdd3ffce26f8fe7#r152777933

Reply:
I assume this is due to permission issues, ping Carl and Nabil in slack on this